### PR TITLE
Linux has added a new capability audit_read

### PR DESCRIPTION
**Pull request by rhatdan**
Docker-DCO-1.1-Signed-off-by: Dan Walsh <dwalsh@redhat.com> (github: rhatdan)

### CODE REVIEW

File: vendor/src/github.com/syndtr/gocapability/capability/enum.go [hunk 1]
```diff
@@ -112,6 +112,8 @@ func (c Cap) String() string {
 		return "wake_alarm"
 	case CAP_BLOCK_SUSPEND:
 		return "block_suspend"
+	case CAP_AUDIT_READ:
```
Comment (vmarmol) [inline vendor/src/github.com/syndtr/gocapability/capability/enum.go]: This is a change on the upstream dependency, we typically send a PR to that.

Comment (rhatdan) [inline vendor/src/github.com/syndtr/gocapability/capability/enum.go]: I sent one to them also.

Comment (vmarmol) [inline vendor/src/github.com/syndtr/gocapability/capability/enum.go]: Thanks :) this LGTM

**Comment by mrunalp** [comment]
@rhatdan This looks good, but the right way to include this will be to have this merged upstream and then update the commit hash here https://github.com/docker/libcontainer/blob/master/update-vendor.sh#L46.

**Comment by rhatdan** [comment]
Well I submitted it upstream.

**Comment by LK4D4** [comment]
syndtr/gocapability#3 for reference

**Comment by rhatdan** [comment]
Upstream has merged.

**Comment by LK4D4** [comment]
@rhatdan Cool! Could you pls update revision in `update-vendor.sh`, run it and commit result here?

**Comment by rhatdan** [comment]
@LK4D4 I would rather wait for @crosbymichael since there were other changes to upstream to stop this from happening again.  Basically allowing us to always drop caps even if the tool does not know about them.  Want to see how he wants to implement it.

**Comment by rhatdan** [comment]
@mrunalp @crosbymichael Not sure how to handle this, since I have requested a pull from upstream gocapability.  

Should I run the update-vendor.sh and then send an updated patch. Or is this something you guys do?

This is why the tests are failing.

**Comment by mrunalp** [comment]
@rhatdan you can update the commit id and run update-vendor.sh and then send the updated patch. Thanks.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**